src/api.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const serverless = require("serverless-http");
+const axios = require("axios");
+
+require("dotenv").config();
+
+const app = express();
+
+const router = express.Router();
+
+router.get("/ready", (req, res) => {
+  res.json({ message: "hello" });
+});
+
+router.post("/ready", async (req, res) => {
+  // try {
+  var reqData = JSON.parse(req.body);
+  var message = `${reqData.repository.owner.login} just pushed a commit with message - '${reqData.head_commit.message}' to <${reqData.repository.name}>`;
+  if (
+    reqData.repository.owner.login &&
+    reqData.head_commit.message &&
+    reqData.repository.name === null
```

Comment:
should be checking all the values used; 
if login is null here, it will not throw 400;